addons/curator-link/addon.json

```diff
@@ -23,16 +37,35 @@
     }
   ],
   "settings": [
+    {
+      "id": "mentions",
+      "name": "Make forum mentions clickable",
+      "type": "boolean",
+      "default": true
+    },
+    {
+      "id": "curator",
+      "name": "Make front page curator cickable",
+      "type": "boolean",
+      "default": true
+    },
     {
       "id": "styleAsNormalText",
       "name": "Style as normal text",
       "type": "boolean",
-      "default": false
+      "default": false,
+      "if": {
+        "settings": { "curator": true }
+      }
```

Comment:
Note: this usage of "if" is valid - you shouldn't change it.
Using "if" with userscripts is the only problematic situation